apps/website/src/components/courses/CourseDirectory.tsx

```diff
@@ -1,15 +1,15 @@
+import type { Course } from '@bluedot/db';
 import {
-  FC,
-} from 'react';
-import {
-  Section,
   ProgressDots,
+  Section,
 } from '@bluedot/ui';
-import type { GetCoursesResponse } from '../../pages/api/courses';
+import {
+  FC,
+} from 'react';
 import { CourseSearchCard } from './CourseSearchCard';
 
 export type CourseDirectoryProps = {
-  courses: GetCoursesResponse['courses'] | undefined;
+  courses?: Course[];
```

Comment:
Maybe this should be inferred from the router output instead.

Reply:
The course pages injects some course information into the document head (I guess for SEO and rich link rendering?).

Because of this I'm going to keep passing the courses as a prop but will update to use router output.